prow/pubsub/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
```

Comment:
Can you add a `area/prow/pubsub` label here and in `label_sync`? This will help you track overall progress here.

Reply:
Done